TicTacToe/GamePlay.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+
+namespace TicTacToe
+{
+    public class GamePlay
+    {
+        private int _player;
+        private GameStatus _gameStatus;
+        private readonly Board _board;
+        private readonly int[,] _boardArray;
+        private bool _givenUp;
+        
+        public GamePlay()
+        {
+            _board = new Board();
+            _boardArray = _board.GetBoard();
+            _player = RandomlyPickPlayerForFirst();
+            _givenUp = false;
+            _gameStatus = GameStatus.Playing;
+        }
+
+        public void Play()
+        {
+            var uiConsole = new UserInputConsole();
```

Comment:
possibly outside the scope of things we are looking at but this looks like a dependency so should be up in the constructor.